Source/WebKit/Shared/RemoteLayerTree/RemoteScrollingUIState.cpp

```diff
@@ -96,4 +96,13 @@ void RemoteScrollingUIState::removeNodeWithActiveUserScroll(WebCore::ScrollingNo
         m_changes.add(Changes::UserScrollNodes);
 }
 
+void RemoteScrollingUIState::clearNodesWithActiveUserScroll()
```

Comment:
Don't you need to call the base class?

Reply:
The other `RemoteScrollingUIState` methods don't seem to call any base class?

Reply:
Sorry, I meant to comment on `RemoteScrollingTree::clearNodesWithUserScrollInProgress()`